src/components/Admin/Module/DalSettings.vue

```diff
@@ -93,18 +109,18 @@ export default {
   data () {
     const systemFieldEncoding = this.module.config.dal.systemFieldEncoding || {}
     const systemFields = [
-      { field: 'id', storeIdent: 'id' },
-      { field: 'namespaceID', storeIdent: 'rel_namespace' },
-      { field: 'moduleID', storeIdent: 'rel_module' },
-      { field: 'revision', storeIdent: 'revision' },
-      { field: 'meta', storeIdent: 'meta' },
-      { field: 'ownedBy', storeIdent: 'owned_by' },
-      { field: 'createdAt', storeIdent: 'created_at' },
-      { field: 'createdBy', storeIdent: 'created_by' },
-      { field: 'updatedAt', storeIdent: 'updated_at' },
-      { field: 'updatedBy', storeIdent: 'updated_by' },
-      { field: 'deletedAt', storeIdent: 'deleted_at' },
-      { field: 'deletedBy', storeIdent: 'deleted_by' },
+      { field: 'id', storeIdent: 'id', disabled: true, groupType: ['all'] },
+      { field: 'namespaceID', storeIdent: 'rel_namespace', disabled: false, groupType: ['all', 'partition'] },
+      { field: 'moduleID', storeIdent: 'rel_module', disabled: false, groupType: ['all', 'partition'] },
+      { field: 'revision', storeIdent: 'revision', disabled: false, groupType: ['all', 'extras'] },
```

Comment:
If 'all' is in every single field then you don't need to explicitly add it, plus if you rename the groupType to group it can now become just a string